solc/jsonCompiler.cpp

```diff
@@ -305,6 +307,11 @@ static string s_outputBuffer;
 
 extern "C"
 {
+extern char const* license()
+{
+	/// TOOD: include the copyright information on the top.
```

Comment:
Is this a todo for this pull request? The license is different from the copyright, we can combine them in the commandline interface.